### PR TITLE
Edit manual testing for DG

### PR DESCRIPTION
**Pull request by Zhi-You**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -247,7 +247,7 @@ After a quiz has started, by making use of `QuizQuestionsManager`'s `getNextQues
 
 As mentioned earlier in the section on the design of the quiz system, a `QuizQuestionsManager` object will randomly select the indicated number of questions from the list of questions in the `Topic` object, and these will form the quiz questions for the user.
 
-The sequence diagram below shows how `QuizQuestionsManager` is implemented to achieve this for the scenario where the user indicates that he wants to attempt 5 questions from the topic on OOP, which translates to the `setQuizQuestions(5, questionsInTopic)` call:
+The sequence diagram below shows how `QuizQuestionsManager` is implemented to achieve this for the scenario where the user indicates that he wants to attempt 5 questions from the topic on [OOP](#34-glossary), which translates to the `setQuizQuestions(5, questionsInTopic)` call:
```
Comment (kstonekuan) [inline docs/DeveloperGuide.md:250]: Good catch for glossary

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -580,4 +580,30 @@ The following are a few testcases to try out.
...
+   1. Test case: quiz n/2 t/oop a/10
+      Expected: Quiz will not start. Error message will show and the ideal format structure will be displayed to user.
+   2. Test case: quiz n/2 t/oop s/0
+      Expected: Quiz will not start. Error message will show to prompt user to give a valid time limit.
+   3. Test case: quiz n/2 t/none s/5
+      Expected: Quiz will not start. Error message will tell user that the topic does not exist in E-Duke-8 and where they can find the available topics.
+   4. Test case: quiz n/0 t/oop s/5
+      Expected: Quiz will not start. Error message will tell user to specify more than 0 quiz questions.
+   4. Other ways to start quiz in wrong format: Use the quiz command but give insufficient inputs or invalid inputs
+      Expected: Similar to previous.
```
Comment (kstonekuan) [inline docs/DeveloperGuide.md]: I think you have to use <br> before every "Expected" so that its on the next line

Comment (Zhi-You) [reply docs/DeveloperGuide.md]: Alright! Fixed, thanks!

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -580,4 +580,30 @@ The following are a few testcases to try out.
...
     5. Other ways to corrupt the data files: Remove other keys or mess up the JSON format (remove the outer array) <br>
-       Expected: Similar to previous.
\ No newline at end of file
+       Expected: Similar to previous.
+       
+#### 3.5.2. Starting a quiz
+
+1. Giving quiz commands with different input formats
+    1. Test case: `quiz n/2 t/oop s/10` <br>
+       Expected: Starts a quiz successfully ith 2 questions from the topic of [OOP](#34-glossary) and a time limit for 10 seconds for each question.
```
Comment (kstonekuan) [inline docs/DeveloperGuide.md:589]: Looks good

**Review by kstonekuan — APPROVED** [review]
lgtm

**Review by anqi20 — APPROVED** [review]
Looks good! Thank you for adding the manual testing.

**Pull request merged**